.github/workflows/release.yml

```diff
@@ -11,24 +11,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: '3.x'
-      - name: Install poetry
-        run: |
-          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
-          echo "PATH=$HOME/.poetry/bin:$PATH" >> $GITHUB_ENV
-      - name: Build distributions
-        run: |
-          poetry build
-      - name: Upload distribution artifacts
-        uses: actions/upload-artifact@v2
+      - name: "Build package using Poetry and store result."
+        uses: chaoss/grimoirelab-github-actions/build@master
         with:
-          name: toolkit-dist
-          path: dist
-  
+          artifact-name: grimoire-dist
```

Comment:
How about changing the `artifact-name`
```
          artifact-name: toolkit-dist
```

If we want to have the same name across the grimoirelab repos, then I think we need to also update the [artifact name in the `tests` step](https://github.com/chaoss/grimoirelab-toolkit/blob/e5a8ad1d94da569f8c94c92b98cd027103c882aa/.github/workflows/release.yml#L43).

Reply:
Yes, toolkit-dist its a better name :ok_hand: